_src/src/main.js

```diff
@@ -1,15 +1,39 @@
 // The Vue build version to load with the `import` command
 // (runtime-only or standalone) has been set in webpack.base.conf with an alias.
 import Vue from 'vue'
+import VueI18n from 'vue-i18n'
 import App from './App'
 import router from './router'
+import utils from './utils.js'
 
 Vue.config.productionTip = false
 
+Vue.use(VueI18n);
+
+// i18n work
+const supportLanguage = [ "zh-CN", "default" ];
```

Comment:
Why `default`, not `en`?

Reply:
First I think en is a gobal language, so I use the `default`. `en` may better, thanks 😄